composer.json

```diff
@@ -14,7 +14,7 @@
         "mockery/mockery": "^1.0",
         "squizlabs/php_codesniffer": "^3.3",
         "psalm/plugin-laravel": "^1.1",
-        "zendframework/zend-diactoros": "^1.0"
+        "laminas/laminas-diactoros": "^2.0"
```

Comment:
For now, let's stay on `^1.0` and create a separate branch to test `^2.0` on an actual application before we actually change it.

Reply:
Done